docs/data/sources/google-tag-manager.md

```diff
@@ -1,12 +1,51 @@
+Use the official Google Tag Manager templates for Amplitude Analytics for data collection. Click on the template name below to access the corresponding tutorial video. 
 
-|Template|Description|
-|---|-----------|
-|[Client-side tag](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-browser-sdk-gtm-template)|The tag uses the Amplitude Marketing Analytics SDK for data collection.|
-|[Client-side tag (legacy)](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-gtm-template)| The tag uses the Amplitude JavaScript SDK for data collection. |
-|[Server-side tag](https://tagmanager.google.com/gallery/#/owners/amplitude/templates/amplitude-server-gtm-template)| The tag uses the Amplitude HTTP API for data collection. |
\ No newline at end of file
+|Template|Type|Description|
+|---|---|-----------|
+|[Amplitude Analytics Browser SDK](./google-tag-manager-client.md)|Client-side|The tag uses the Amplitude Marketing Analytics SDK for data collection.|
+|[Amplitude Analytics](./google-tag-manager-server.md)|Server-side|The tag uses the Amplitude HTTP V2 API for data collection. |
+|[Amplitude Analytics Legacy](./google-tag-manager-client-legacy.md)|Client-side|The tag uses the Amplitude JavaScript SDK for data collection. |
+
+Amplitude's GTM templates offer a convenient way for developers to set up event tracking in their applications. These pre-configured templates provide a streamlined solution for deploying tracking tags, reducing the time and effort required to set up event tracking. Learn more about [Google Tag Manager Templates](https://developers.google.com/tag-platform/tag-manager/templates).
+
+## Client-side vs server-side
+
+Google Tag Manager (GTM) offers both client-side and server-side templates for tag deployment. GTM client side templates and GTM server side templates differ in the way they execute the tracking plan.
+
+Client-side templates run the tracking plan directly on the user's device and use JavaScript code to manage tags. They serve tags that need to interact with the client environment, such as tracking user actions on a website. However, the client browser capabilities restrict these templates and may not be appropriate for tags that require access to server-side data. 
+
+GTM server-side templates run on the server, not on the client's browser. They control tags with server-side code and serve tags that require access to server-side data, such as monitoring server-side events. These templates offer a higher level of control over tag deployment and can handle complex tag implementations
+
+Use GTM client-side templates to track user interactions on a website. Use server-side templates if you want to track sensitive customer information or complex tags that require server-side processing.
+
+## Getting started
+
+### Install the template
+
+1. [Set up and install Google Tag Manager](https://support.google.com/tagmanager/answer/6103696) if you haven't done so. 
```

Comment:
Consider reworking these instructions to bypass all this browsing stuff and just directly link to the template page, where they can click "Add to Workspace". 

Reason: use the fewest clicks and steps you can. Even if it's not the way product wants you to use the tool 😂 

example: 
1. Set up and install .....
2. Choose a template: Link, Link, Link. 
3. Click **Add to Workspace**

Reply:
Changed it to the following:

> 
> 1. Go to [Tag Manager](https://tagmanager.google.com/) and select your container or [set up a new one](https://support.google.com/tagmanager/answer/6103696).
> 2. Open the **Templates** tab and click **Search Gallery** in the **Tag Templates** section.
> 3. Type "Amplitude" in the search bar and select the Amplitude GTM Template.
> 4. Add the template to your workspace by clicking "Add to Workspace".
>

Reply:
Also shorten the guide for creating a tag below this section